### DISCUSSION TITLE
Checkbox option?

### DISCUSSION THREAD

**Discussion by Digitalduckinc** [glushchenko/fsnotes#1765] [General]
Ist time on Github so please accept my apologies if this was addressed already or if I am in the wrong location. I did a search and could not find any references to this topic. I love this app and use it on my desktop mac to stay organized. One of my favourite features was the ability to add checkboxes to an existing list but I don't see that option in v6.9.5 build 644. My checkboxes are still visible in previous notes but there is no longer a right-shift option to add checkboxes. Any help greatly appreciated!

**Comment by glushchenko** [comment]
⌘-t
